dukconfig/build_dist.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Please use `#!/bin/sh` instead of GNU/Linux-specific paths like `/bin/bash`. You're not even using any bash-isms in this tiny script :)